src/SmartTransactionsController.test.ts

```diff
@@ -450,29 +502,53 @@ describe('SmartTransactionsController', () => {
   });
 
   describe('submitSignedTransactions', () => {
-    beforeEach(() => {
-      // eslint-disable-next-line jest/prefer-spy-on
-      smartTransactionsController.checkPoll = jest.fn(() => ({}));
-    });
-
     it('submits a smart transaction with signed transactions', async () => {
-      const signedTransaction = createSignedTransaction();
-      const signedCanceledTransaction = createSignedCanceledTransaction();
-      const submitTransactionsApiResponse = createSubmitTransactionsApiResponse(); // It has uuid.
-      nock(API_BASE_URL)
-        .post(`/networks/${ethereumChainIdDec}/submitTransactions`)
-        .reply(200, submitTransactionsApiResponse);
-
-      await smartTransactionsController.submitSignedTransactions({
-        signedTransactions: [signedTransaction],
-        signedCanceledTransactions: [signedCanceledTransaction],
-        txParams: signedTransaction,
-      });
-
-      expect(
-        smartTransactionsController.state.smartTransactionsState
-          .smartTransactions[CHAIN_IDS.ETHEREUM][0].uuid,
-      ).toStrictEqual('dP23W7c2kt4FK9TmXOkz1UM2F20');
+      const chainIdHex = '0x64';
```

Comment:
I updated this test so that it didn't rely on the `beforeEach` above — it creates its own instance of SmartTransactionsController so that we can override the `config` — and also moves the UUID which was previously set in another function into this test so it's easier to see its relationship with the expected value.